v21.1/restore.md

```diff
@@ -102,7 +102,7 @@ You can restore:
 - All [tables](create-table.html) (which automatically includes their [indexes](indexes.html))
 - All [views](views.html)
 
-An object's historical data is not preserved upon `RESTORE`. This means that if you issue an `AS OF SYSTEM TIME` query on a restored object, the query will fail or the response will be incorrect because there is no historical data to query. For example, if you restore a table at `2022-07-13 10:38:00`, it is not then possible to read or [back up](backup.html) that table at `2022-07-13 10:37:00` or earlier. This is also the case for backups with [`revision_history`](backup.html#with-revision-history) that might try to initiate a revision start time earlier than `2022-07-13 10:38:00`.
+`RESTORE` will only restore the latest value of an object (table, database, cluster), or the latest value as per an [`AS OF SYSTEM TIME` restore](#as-of-system-time). That is, a restore will not include historical data even if you ran your backup with [`revision_history`](backup.html#with-revision-history). This means that if you issue an `AS OF SYSTEM TIME` query on a restored object, the query will fail or the response will be incorrect because there is no historical data to query. For example, if you restore a table at `2022-07-13 10:38:00`, it is not then possible to read or [back up](backup.html) that table at `2022-07-13 10:37:00` or earlier. This is also the case for backups with `revision_history` that might try to initiate a revision start time earlier than `2022-07-13 10:38:00`.
 
```

Comment:
nit: I'd change "latest value on" to "latest data in". Other than that. looks good to me!

Reply:
Cool, thanks for re-reviewing so quickly and your suggestions Michael! I'll make that change now.